src/SupervisionSuccess/components/__tests/ChangeInRevocations.test.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
```

Comment:
Nit: I think you meant for the sub-directory to be `__tests__` not `__tests`, or am I mistaken?